### PR TITLE
Add resources for "Deep Learning"

### PR DESCRIPTION
**Pull request by Nish-19**
I have added a 3blue1brown deep learning introductory videos (neural networks) playlist.

{If you have added a resource, please explain your experience with the resource in context of why you have added it.}

This video series is very useful for beginners to understand what neural networks are, and understand their behavior. I found it useful myself when I was starting, as it gives an apt pictorial representation of the neural networks along with their working explained through an illustrative example. Overall I feel that it is a must watch for people who want to understand the basics.

--------

### By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/learn-for-free/blob/master/CONTRIBUTING.md) and made sure my pull request follows it, specifically the part about promotions and markdown format.
- [x] Any resources I am adding in this PR have been used by me and I can personally vouch for their quality.
- [x] This pull request has a descriptive title. For example, `Add [Resource Name]` or `Add [Topic Name]`, not `Added new entries` or `update`.
- [x] I have double checked that I have only updated the minimum files that are required by this PR.

### CODE REVIEW

File: data_science.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ This file contains resources to learn about Data Science and its sub fields.
...
+
+## Deep Learning
+
+### Neural Networks by 3Blue1Brown
+
+A playlist containing 4(four) videos about neural networks and their working. It gives a mathematical intution about neural networks and concepts like Gradient Desent and Backpropogation. 
+
+Website - https://www.youtube.com/playlist?list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi
+
+![3blue1brown](https://user-images.githubusercontent.com/41947720/66421378-6161ce00-ea25-11e9-9128-10473cd5ccb4.png)
```
Comment (aviaryan) [inline data_science.md]: Please link the image to the website.

File: data_science.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ This file contains resources to learn about Data Science and its sub fields.
...
-* Deep Learning
+* [Deep Learning](#deep-learning)
 * Natural Language Processing 
 * Reinforcement Learning
+
+## Deep Learning
+
+### Neural Networks by 3Blue1Brown
+
+A playlist containing 4(four) videos about neural networks and their working. It gives a mathematical intution about neural networks and concepts like Gradient Desent and Backpropogation.
```
Comment (aviaryan) [inline data_science.md]: 'Descent' is spelled wrong. Please double-check for spelling errors.

File: data_science.md (hunk 1 above)
Comment (Nish-19) [reply data_science.md]: @aviaryan, thanks for the instruction, I have made the requested change.

File: data_science.md (hunk 2 above)
Comment (Nish-19) [reply data_science.md]: @aviaryan, thanks for the instruction, I have made the requested change.

**Pull request merged**